tests/GasModelSpec.hs

```diff
@@ -7,6 +7,7 @@
 module GasModelSpec (spec) where
 
 import Test.Hspec
+import Test.Hspec.Core.Spec (sequential)
```

Comment:
There is already a fix for this on master. This line is not needed with `hspec-2.10.10`.